editor/src/components/canvas/canvas-strategies/strategies/set-border-radius-strategy.tsx

```diff
@@ -207,12 +214,38 @@ function borderRadiusFromElement(
   }
 
   const fromProps = borderRadiusFromProps(jsxElement.props)
+  const measurementsNonZero = AllSides.some((c) => (renderedValueSides[c] ?? 0) > 0)
+
+  const elementIsIntrinsicElementOrScene = foldEither(
+    () => false,
+    (e) =>
+      isJSXElement(e) &&
+      (isIntrinsicElement(e.name) || jsxElementNameEquals(e.name, jsxElementName('Scene', []))),
+    element.element,
+  )
+
+  if (
+    !elementIsIntrinsicElementOrScene &&
+    !shouldShowControls({
```

Comment:
I think `!(elementIsIntrinsicElementOrScene || !shouldShowControls` would be easier on my brain

Reply:
agreed to do it as a separate PR